jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -23,7 +23,17 @@
      (setq moveit-robot (instance fetch-robot :init))
      (send self :set-moveit-environment (instance fetch-moveit-environment :init :robot moveit-robot))
      ))
-  (:angle-vector-raw (&rest args) (send-super* :angle-vector args))
+  (:angle-vector-raw (av &optional (tm 3000) &key (devide 4) &rest args)
+   (let (avs tms (minjerk (instance minjerk-interpolator :init)))
+     (send minjerk :reset
+           :position-list (list (send self :state :potentio-vector :wait-until-update t) av)
+           :time-list (list tm))
+     (send minjerk :start-interpolation)
+     (send minjerk :pass-time (/ tm devide))
```

Comment:
if you fo `(send minjerk :pass-time time)` for the first time, it returns `angle-vector` at time `0`s.
Is this a bug or a designed return? @k-okada 

```
1.irteusgl$ (setq minjerk (instance minjerk-interpolator :init))
#<minjerk-interpolator #X591c800>
2.irteusgl$ (send minjerk :reset :position-list (list #f(0 0 0) #f(0 10 20)) :time-list (list 5000))
(#f(0.0 0.0 0.0) #f(0.0 0.0 0.0))
3.irteusgl$ (send minjerk :start-interpolation)
t
4.irteusgl$ (send minjerk :pass-time 1000)
#f(0.0 0.0 0.0)
5.irteusgl$ (send minjerk :pass-time 1000)
#f(0.0 0.5792 1.1584)
6.irteusgl$ (send minjerk :pass-time 1000)
#f(0.0 3.1744 6.3488)
7.irteusgl$ (send minjerk :pass-time 1000)
#f(0.0 6.8256 13.6512)
8.irteusgl$ (send minjerk :pass-time 1000)
#f(0.0 9.4208 18.8416)
9.irteusgl$ (send minjerk :pass-time 1000)
#f(0.0 10.0 20.0)
10.irteusgl$ (send minjerk :pass-time 1000)
nil
```